### ISSUE TITLE
Vulkan render is transparent in Linux

### ISSUE THREAD

**Issue by BogdanTheGeek** [yuzu-emu/yuzu#3476]
These screenshots are from a previous build but the same issue is happening with the current mainline version.
I have my specs and other system info in this issue:
https://github.com/yuzu-emu/yuzu/issues/3390
Let me know what else I should provide in order to help.
![Screenshot from 2020-02-09 18-42-15](https://user-images.githubusercontent.com/8824186/75631591-20dd3980-5bec-11ea-849c-a5d95bbc17a1.png)
![Screenshot from 2020-02-09 18-42-22](https://user-images.githubusercontent.com/8824186/75631593-23d82a00-5bec-11ea-8b87-7d1bd85d9baf.png)

**Comment by luziferius** [comment]
Which renderer device do you use? You can choose the GPU renderer in the Yuzu settings when using Vulkan.
It works fine for me on KDE 5 with an NVidia GPU. Did you maybe select the integrated Intel GPU?
Which window manager do you use?

My stats for comparison, running Kubuntu 19.10:
```
CPU: 8-Core AMD Ryzen 7 3700X (-MT MCP-) speed/min/max: 2195/2200/3600 MHz Kernel: 5.6.0-050600rc4-generic x86_64 Up: 4h 26m 
Mem: 5416.9/32095.4 MiB (16.9%) Storage: 21.83 TiB (41.0% used) Procs: 432 Shell: bash 5.0.3 inxi: 3.0.36
Graphics:  Device-1: NVIDIA GP104 [GeForce GTX 1070] driver: nvidia v: 440.64 
           Display: x11 server: X.Org 1.20.5 driver: nvidia unloaded: fbdev,modesetting,nouveau,vesa 
           resolution: 1680x1050~60Hz, 1680x1050~60Hz 
           OpenGL: renderer: GeForce GTX 1070/PCIe/SSE2 v: 4.6.0 NVIDIA 440.64 
```

**Comment by BogdanTheGeek** [comment]
Its with vulkan and the nvidia gpu, open gl is fine now.

**Comment by exalented** [comment]
I have not experienced this issue in yuzu ever.
-Arch
-i3wm
-1080

**Comment by BogdanTheGeek** [comment]
> Are you sure it isn't a problem with the way you have gnome setup?
What could be wrong?

**Comment by gusthe** [comment]
I also have this issue. It happens with both nvidia(1050ti) and amd(rx470) cards. I'm running arch with xfce. The only ways to solve it is to turning off composting(but then I get tearing) or make my desktop black and have no windows behind yuzu, .

**Comment by v1993** [comment]
Does this happen with any other vulkan application, like vkcube/vkcubepp demo? This isn't an issue for me on KDE, Nvidia proprietary driver.

**Comment by gusthe** [comment]
vkcube and vkcubepp both work, and are not transparent.

**Comment by v1993** [comment]
For any further research: yuzu does properly set swapchain's `compositeAlpha` to `VK_COMPOSITE_ALPHA_OPAQUE_BIT_KHR`, so the issue is somewhere else.

**Comment by popsUlfr** [comment]
Ok, this is a weird one. I'm seeing this issue myself only on Breath of the Wild. Astral Chain with the Vulkan renderer for instance does not exhibit this behaviour.
Whatever is in the background is bleeding through.

Disabling compositing (Shift+Alt+F12) fixes the issue on Breath of the Wild.

What I notice on BOTW is that when starting the game, after the switch launching screen the black screen flashes and becomes transparent. 

My specs:
```
OS: Arch Linux
Kernel: 5.13.9-zen1-1-zen
DE: KDE Plasma 5.22.4
CPU: AMD Ryzen 9 5900X
GPU: NVIDIA GeForce GTX 980 Ti (470.63.01)
```

**Comment by v1993** [comment]
If different games result in different transparency I'd assume it have to do with `compositeAlpha` somehow breaking (compositors misinterpret/ignore it?). Sadly, I don't have BotW, so I can't test on it.

**Comment by popsUlfr** [comment]
I tried to look into vulkan validation errors with 
```
export VK_INSTANCE_LAYERS=VK_LAYER_LUNARG_api_dump:VK_LAYER_KHRONOS_validation
```
but there's a constant stream of errors and I'm not sure what could helpful in all of it. The first one looks like this

```
Thread 0, Frame 0:
vkCmdBindVertexBuffers2EXT(commandBuffer, firstBinding, bindingCount, pBuffers, pOffsets, pSizes, pStrides) returns void:
    commandBuffer:                  VkCommandBuffer = 0x56099a760e18
    firstBinding:                   uint32_t = 0
    bindingCount:                   uint32_t = 1
    pBuffers:                       const VkBuffer* = 0x56099af70ac8
        pBuffers[0]:                    const VkBuffer = 0xd60fd30000001225
    pOffsets:                       const VkDeviceSize* = 0x7f42e09fa320
        pOffsets[0]:                    const VkDeviceSize = 512
    pSizes:                         const VkDeviceSize* = 0x7f42e09fa328
        pSizes[0]:                      const VkDeviceSize = 128
    pStrides:                       const VkDeviceSize* = 0x7f42e09fa330
        pStrides[0]:                    const VkDeviceSize = 32

Thread 0, Frame 0:
vkCmdBindVertexBuffers2EXT(commandBuffer, firstBinding, bindingCount, pBuffers, pOffsets, pSizes, pStrides) returns voidUNASSIGNED-GeneralParameterError-RequiredParameter(ERROR / SPEC): msgNum: -1711571459 - Validation Error: [ UNASSIGNED-GeneralParameterError-RequiredParameter ] Object 0: handle = 0x56099afede68, type = VK_OBJECT_TYPE_DEVICE; | MessageID = 0x99fb7dfd | vkCmdBindVertexBuffers2EXT: required parameter pBuffers[0] specified as VK_NULL_HANDLE
    Objects: 1
        [0] 0x56099afede68, type: 3, name: NULL
VUID-vkCmdBindVertexBuffers2EXT-pBuffers-parameter(ERROR / SPEC): msgNum: 1747617674 - Validation Error: [ VUID-vkCmdBindVertexBuffers2EXT-pBuffers-parameter ] Object 0: handle = 0x7f42b0000b60, type = VK_OBJECT_TYPE_INSTANCE; | MessageID = 0x682a878a | Invalid VkBuffer Object 0x0. The Vulkan spec states: pBuffers must be a valid pointer to an array of bindingCount valid VkBuffer handles (https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#VUID-vkCmdBindVertexBuffers2EXT-pBuffers-parameter)
    Objects: 1
        [0] 0x7f42b0000b60, type: 1, name: NULL
:
    commandBuffer:                  VkCommandBuffer = 0x56099a760e18
    firstBinding:                   uint32_t = 1
    bindingCount:                   uint32_t = 1
    pBuffers:                       const VkBuffer* = 0x56099af70af8
        pBuffers[0]:                    const VkBuffer = 0
    pOffsets:                       const VkDeviceSize* = 0x7f42e09fa320
        pOffsets[0]:                    const VkDeviceSize = 0
    pSizes:                         const VkDeviceSize* = 0x7f42e09fa328
        pSizes[0]:                      const VkDeviceSize = 18446744073709551615
    pStrides:                       const VkDeviceSize* = 0x7f42e09fa330
        pStrides[0]:                    const VkDeviceSize = 0
```

I tried capturing the Vulkan API calls with GFXReconstruct `gfxrecon-capture -o yuzu.gfxr yuzu` but trying to replay it with `gfxrecon-replay yuzu*.gfxr` results in

```
[gfxrecon] INFO - Replay adjusted the vkGetPhysicalDeviceToolPropertiesEXT array count: capture count = 1, replay count = 0
[gfxrecon] FATAL - API call vkQueuePresentKHR returned error value VK_ERROR_OUT_OF_DATE_KHR that does not match the result from the capture file: VK_SUCCESS.  Replay cannot continue.
Replay has encountered a fatal error and cannot continue: a surface has changed in such a way that it is no longer compatible with the swapchain
```

I tried to swap `VK_COMPOSITE_ALPHA_OPAQUE_BIT_KHR` with `VK_COMPOSITE_ALPHA_INHERIT_BIT_KHR` in https://github.com/yuzu-emu/yuzu/blob/master/src/video_core/renderer_vulkan/vk_swapchain.cpp#L170 but that didn't change anything on my end.

**Comment by liamwhite** [comment]
Fixed by #7410
